needlr/dataactivator/reflex/reflex.py

```diff
@@ -165,16 +167,17 @@ def ls(self, workspace_id:uuid.UUID, continuation_token: str) -> Iterator[Reflex
             Reference:
                 [List Reflexes](https://learn.microsoft.com/en-us/rest/api/fabric/reflex/items/list-reflexes?tabs=HTTP)
             """
+            flag = f'?continuationToken={continuation_token}' if continuation_token else ''
```

Comment:
@emmynguyen I don't think the continuationToken flag / check is necessary

https://github.com/microsoft/needlr/blob/a7a28b14817fa9354bece22a2995ecdffc864391/needlr/_http.py#L117

ContinuationToken is already present in _get_http_paged